client/packages/system/src/Vaccination/api/operations.generated.ts

```diff
@@ -258,17 +250,17 @@ export const VaccinationDetailFragmentDoc = gql`
       lastName
     }
     given
+    givenStoreId
+    item {
+      id
+      name
+    }
     stockLine {
       id
-      itemId
-      itemName
       batch
     }
     invoice {
       id
-      store {
-        id
-      }
```

Comment:
Given store id more reliable than invoice, which would be null everywhere except giving store